pkg/utils/errors.go

```diff
@@ -18,7 +18,7 @@ func ParseErrorToString(err interface{}) string {
 }
 
 func IsNetworkError(err error) bool {
-	networkErrors := []string{"network error", "connection refused", "no such host", "i/o timeout", "server misbehaving"}
+	networkErrors := []string{"network error", "connection refused", "no such host", "i/o timeout", "server misbehaving", "blocked"}
```

Comment:
Did you test it out? I think it won't work because of case sensitivity

Reply:
sorry this is a little premature :sweat_smile: was testing out the Github CLI and raised this before I meant to, I'll close it and update once I have everything working